docs/src/code/building-linuxcnc.adoc

```diff
@@ -291,6 +292,14 @@ platform to build for.  The normal values for this argument are:
     You must be running an RTAI kernel on your build machine for this
     to work!
 
+`--help`::
+
+    presets a very short summary of extra options
```

Comment:
Type - should be "Presents".

Reply:
Done. It unfortunately took me another iteration to recognize that the sentence should start with a capital letter. Also  fixed for the following section.